### PR TITLE
Add examples of instance recovery

### PR DESCRIPTION
**Pull request by gitzhou**

### CODE REVIEW

**Comment by gitzhou** [comment]
wait for next beta version

File: tests/testnet/counterFromTx.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    return tx
+}
+
+async function callIncrementOnChain(
+    tx: bsv.Transaction,
+    atOutputIndex = 0
+): Promise<bsv.Transaction> {
+    // recover instance from tx
+    const instance = Counter.fromTx(tx, atOutputIndex)
+    console.log('Recovered')
```
Comment (zhfnjust) [inline tests/testnet/counterFromTx.ts]: remove

File: tests/testnet/p2pkhFromTx.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    console.log('P2PKH contract deployed: ', deployTx.id)
+}
+
+async function call() {
+    // recover instance from tx
+    const p2pkh = P2PKH.fromTx(deployTx, atOutputIndex)
+
+    await p2pkh.connect(getDefaultSigner())
+
+    console.log('Recovered')
```
Comment (zhfnjust) [inline tests/testnet/p2pkhFromTx.ts]: remove

**Review by zhfnjust — APPROVED** [review]
Approve

**Pull request merged**